sparse/tests/test_backends.py

```diff
@@ -69,19 +70,116 @@ def test_asarray(backend, format, order):
 
 
 @pytest.mark.parametrize("format, order", [("csc", "F"), ("csr", "C"), ("coo", "F"), ("coo", "C")])
-def test_scipy_sparse_dispatch(backend, format, order):
+def test_scipy_sparse_linalg_dispatch(backend, format, order):
     x = np.eye(10, order=order) * 2
     y = np.ones((10, 1), order=order)
+    x_pydata = sparse.asarray(x, format=format)
+    y_pydata = sparse.asarray(y, format="coo")
+    # x_sp = sps.coo_matrix(x)
+    # y_sp = sps.coo_matrix(y)
 
-    x_sp = sparse.asarray(x, format=format)
-    y_sp = sparse.asarray(y, format="coo")
-
-    actual = splin.spsolve(x_sp, y_sp)
+    # spsolve
+    actual = splin.spsolve(x_pydata, y_pydata)
     expected = np.linalg.solve(x, y.ravel())
-
     assert_almost_equal(actual, expected)
 
-    actual = splin.inv(x_sp)
+    # inv
+    actual = splin.inv(x_pydata)
     expected = np.linalg.inv(x)
-
     assert_almost_equal(actual.todense(), expected)
+
+    # NOTE: https://github.com/scipy/scipy/pull/20759
+    # norm
+    # actual = splin.norm(x_pydata)
+    # expected = sp.linalg.norm(x)
+    # assert_almost_equal(actual, expected)
+
+    # NOTE: https://github.com/scipy/scipy/pull/20759
+    # lsqr
+    # actual_x, _ = splin.lsqr(x_pydata, y)[:2]
+    # expected_x, _ = sp.linalg.lstsq(x, y)[:2]
+    # assert_almost_equal(actual_x, expected_x.ravel())
+
+    # NOTE: https://github.com/scipy/scipy/pull/20759
+    # eigs
+    # actual_vals, _ = splin.eigs(x_pydata, k=3)
+    # expected_vals, _ = splin.eigs(x_sp, k=3)
+    # assert_almost_equal(actual_vals, expected_vals)
```

Comment:
Let's parametrize the test over functions, and mark these as skips

Reply:
Done!